main.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import os
+from pathlib import Path
+from typing import List
+import mediapipe as mp
+import cv2
+
+# IMAGE_EXTENSIONS = ['png', 'jpg']
+IMAGE_EXTENSIONS = ['mp4']
+DEFAULT_BASE_DIR = 'videos'
+POSE_INIT = mp.solutions.pose.Pose(
+    static_image_mode=False,  # change to False of was video stream
+    model_complexity=1,
+    smooth_landmarks=True,
+    min_detection_confidence=0.5,
+    min_tracking_confidence=0.5
+)
+
+DRAWING = mp.solutions.drawing_utils
+
+
+def detect_pose(frame,pose):
+    frame_rgb = cv2.cvtColor(frame,cv2.COLOR_BGR2RGB)
+    result = pose.process(frame_rgb)  # start detecting the keypoints
+    return result
+
+def draw_pose_on_frame(frame, detected_frame,  drawing):
+    if detected_frame.pose_landmarks:
+        DRAWING.draw_landmarks(
+            image=frame,
+            landmark_list=detected_frame.pose_landmarks,
+            connections=mp.solutions.pose.POSE_CONNECTIONS,
+            landmark_drawing_spec=DRAWING.DrawingSpec(
+                color=(250, 250, 250),
```

Comment:
Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.